### PR TITLE
dist/ami: use prebuilt rpms on --localrpm

### PR DESCRIPTION
**Pull request by syuu1228**
We made --localrpm option to automatically build rpms from sourcecode,
but we actually use the option to produce AMI using prebuilt rpm on our
CI.
To simplified the script, and to prevent accsidently start rpm build
in the script, drop rpm build part.

### CODE REVIEW

File: dist/ami/build_ami.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -e
 
-./SCYLLA-VERSION-GEN
+./SCYLLA-VERSION-GEN > /dev/null 2>&1
```
Comment (avikivity) [inline dist/ami/build_ami.sh]: Why this change?

Comment (syuu1228) [reply dist/ami/build_ami.sh]: @avikivity I thought we don't need to print version number on the beginning of build_ami.sh, but the change is not related with PR so dropped the change.
Pushed new version: https://github.com/scylladb/scylla/pull/5554/files
